### PR TITLE
Fix couple instances of details panel to element inspector 

### PR DESCRIPTION
**Pull request by nidasid**
- [ ] joeferner
- [ ] diegogrz
- [ ] mwizeman sfeng88
- [ ] joeybrk372 rygim jharwig EvanOxfeld

Testing Instructions: verify details panel is now element inspector 

Points of Regression: you shouldn't see `details panel`

CHANGELOG
Documentation: Change `details panel` to `element inspector`

### CODE REVIEW

File: archetype/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
     <groupId>org.visallo</groupId>
     <artifactId>visallo-plugin-archetype</artifactId>
-    <version>3.1-SNAPSHOT</version>
+    <version>3.1.0-RC2-SNAPSHOT</version>
```
Comment (sfeng88) [inline archetype/pom.xml]: this should be 3.1-SNAPSHOT

File: archetype/src/main/resources/META-INF/maven/org.visallo/visallo-plugin-archetype/pom.xml [hunk 2]
```diff
@@ -6,6 +6,6 @@
 
     <groupId>org.visallo</groupId>
     <artifactId>visallo-plugin-archetype</artifactId>
-    <version>3.1-SNAPSHOT</version>
+    <version>3.1.0-RC2-SNAPSHOT</version>
```
Comment (sfeng88) [inline archetype/src/main/resources/META-INF/maven/org.visallo/visallo-plugin-archetype/pom.xml]: same as above

File: archetype/src/main/resources/archetype-resources/pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.visallo</groupId>
         <artifactId>visallo-root</artifactId>
-        <version>3.1-SNAPSHOT</version>
+        <version>3.1.0-RC2-SNAPSHOT</version>
```
Comment (sfeng88) [inline archetype/src/main/resources/archetype-resources/pom.xml]: same as above

File: archetype/src/main/resources/archetype-resources/pom.xml [hunk 4]
```diff
@@ -16,7 +16,7 @@
     <properties>
         <java.source.version>1.8</java.source.version>
         <java.target.version>1.8</java.target.version>
-        <visallo.version>3.1-SNAPSHOT</visallo.version>
+        <visallo.version>3.1.0-RC2-SNAPSHOT</visallo.version>
```
Comment (sfeng88) [inline archetype/src/main/resources/archetype-resources/pom.xml]: same as above

**Comment by nidasid** [comment]
@sfeng88 Closing this and opened https://github.com/visallo/visallo/pull/1064

**Pull request closed without merging**